content/index.fr.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Vue d'ensemble des standards de l'accessibilité"
+permalink: /fundamentals/fr
+ref: /fundamentals/
+lang: fr
+last_updated: 2020-12-28
+translators:
+- name : "Sofia Ahmed"
+contributors:
+- name : "Sandra Velarde Gonzalez (ETNIC)"
+github:
+  repository: w3c/wai-fundamentals-overview
+  path: content/index.fr.md
+footer: > # Text in footer in HTML
+  <p><strong>Date :</strong> Mise à jour : 19 août 2020.</p>
+  <p><strong>Rédactrice :</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Développé par le Groupe de travail Éducation et Promotion (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+
+---
+
+Les ressources suivantes contiennent des introductions à l'accessibilité :
+
+[[Introduction à l'accessibilité Web]](/fundamentals/accessibility-intro/)
+:   Introduit à l'accessibilité Web et renvoie à des ressources supplémentaires.
+    -   [[Vidéo : introduction à l'accessibilité Web et aux standards du W3C (4 minutes)]](/videos/standards-and-benefits/) .
+
+[[Éléments essentiels de l'accessibilité Web]](/fundamentals/components/)
+:   Montre comment l'accessibilité Web dépend de l'association entre les nombreuses composantes du développement Web et de l'interaction utilisateur, et le lien entre les différentes règles de la WAI : les Règles pour l'accessibilité des contenus Web ([WCAG](/standards-guidelines/wcag/), les Règles d'accessibilité pour les outils d'édition ([ATAG](/standards-guidelines/atag/) et les Règles pour l’accessibilité des agents utilisateurs ([UAAG](/standards-guidelines/uaag/).  
+
+[[Principes d'accessibilité]](/fundamentals/accessibility-principles/)
+:   Aide les développeurs, les designers, et tout autre intervenant à comprendre les principes pour créer des sites et des applications Web, des navigateurs, et d'autres outils accessibles. Fournit des liens vers les standards internationaux de l'Initiative pour l'accessibilité du Web (WAI) du W3C et vers une page dédiée aux [témoignages d'utilisateurs du Web](/people-use-web/user-stories/).
+
+[[L'accessibilité Web illustrée : explorer l'impact et les bénéfices pour tous]](/perspective-videos/)
```

Comment:
English : Web Accessibility Perspectives Videos: Explore the Impact and Benefits for Everyone
French translation: L’accessibilité Web illustrée : explorer l’impact et les bénéfices pour tous (en anglais)

The English specifically refers to the videos perspectives whereas the French version refers generally to web accessibility illustrated. 

To be more accurate, I would propose changing the French from "L'accessibilité Web illustrée" to  "Vidéos sur les perspectives de l'accessibilité du Web.

Reply:
Yes. And I'm seeing lots of people use them to show the broader benefits for everyone.

In case it helps, the original purpose and objectives are here : https://www.w3.org/WAI/EO/wiki/Showcase_Examples_with_Videos/Requirements_Analysis

Reply:
+1